source/server/spongineer/commands.rst

```diff
@@ -26,26 +26,27 @@ Sponge
 The following commands are available to players with operator status (or the correct permission node) on servers powered
 by Sponge.
 
-====================  ========================================  ======================
-Command               Description                               Permission
-====================  ========================================  ======================
-/sponge audit         Forces loading of unloaded classes to     sponge.command.audit
-                      enable mixin debugging.
-/sponge chunks        Prints out the chunk data for a world, a  sponge.command.chunks
-                      dimension, or globally.
-/sponge config        Alters a global, world, or a dimension    sponge.command.config
-                      config.
-/sponge heap          Dumps the JVM heap.                       sponge.command.heap
-/sponge plugins       Lists currently installed plugins.        sponge.command.plugins
-/sponge reload        Reloads the global, world, or dimension   sponge.command.reload
-                      config.
-/sponge save          Saves the global, world, or dimension     sponge.command.save
-                      config.
-/sponge timings       The main command for the timings module.  sponge.command.timings
-/sponge tps           Display ticks per second for each world.  sponge.command.tps
-/sponge version       Prints the Sponge/SpongeAPI versions to   sponge.command.version
-                      the console.
-====================  ========================================  ======================
+======================  ========================================  ======================
+Command                 Description                               Permission
+======================  ========================================  ======================
+/sponge audit           Forces loading of unloaded classes to     sponge.command.audit
+                        enable mixin debugging.
+/sponge chunks          Prints out the chunk data for a world, a  sponge.command.chunks
+                        dimension, or globally.
+/sponge config          Alters a global, world, or a dimension    sponge.command.config
+                        config.
+/sponge heap            Dumps the JVM heap.                       sponge.command.heap
+/sponge plugins         Lists currently installed plugins.        sponge.command.plugins
+/sponge plugins reload  Requests plugins to reload configs.       sponge.command.plugins
```

Comment:
As discussed in #492, it's up to the plugins as to what gets reloaded. I'm not sure if 'reload configs' is fully appropriate for its potential use. It's really more of a callback to the plugins to do 'reloading' stuff.